bitnami/clickhouse-operator/templates/rbac.yaml

```diff
@@ -0,0 +1,218 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.rbac.create }}
+kind: ClusterRole
```

Comment:
Not sure it this may cause errors in the logs, but in some charts where you can decide the namespaces to watch, what we do is create a Role/Rolebinding per namespace. You can see it in cloudnative-pg. However, double check that this does not add noise in the logs

Reply:
TODO

Reply:
done at https://github.com/bitnami/charts/pull/32816/commits/f07d8c5a0c14c4e10078b83f623887f6736a00ff